xdf2fieldtrip.m

```diff
@@ -65,6 +64,43 @@
   end
 end
 
+% Read the EEG stream to get the first time stamp and sampling rate
+EEG_info=[];
+for i=1:length(streams)
+    if (strcmp(streams{i}.info.type,'EEG'))
+        EEG_t1= str2double(streams{i}.info.first_timestamp);
+        EEG_srate=streams{i}.info.effective_srate;
+        EEG_info=[EEG_t1 EEG_srate];
+    end
+end
+
+if isempty(EEG_info)
+  ft_error('no EEG streams were selected');
```

Comment:
do you want an error in case the data only contains non-EEG data, e.g. only eye tracking?